### PR TITLE
Remove redundant script

### PR DESCRIPTION
**Pull request by Cat-Ion**

### CODE REVIEW

File: firmware/configs/sha2017_defconfig [hunk 1]
```diff
@@ -806,7 +806,7 @@ CONFIG_MICROPY_USE_DISPLAY=
 CONFIG_MICROPY_USE_GPS=
 CONFIG_MICROPY_USE_ETHERNET=
 CONFIG_MICROPY_USE_MDNS=
-CONFIG_MICROPY_USE_REQUESTS=y
+CONFIG_MICROPY_USE_REQUESTS=n
```
Comment (renzenicolai) [inline firmware/configs/sha2017_defconfig]: Can you remove this "n" and instead just clear the field?
Also: please separate these changes or rename your PR, pick one :D

**Review by renzenicolai — CHANGES_REQUESTED** [review]

File: firmware/configs/sha2017_defconfig (hunk 1 above)
Comment (Cat-Ion) [reply firmware/configs/sha2017_defconfig]: ... that's weird, I based this on my other PR a while ago with these two commits, and would've expected it to remove them when that's merged. I'll just rebase. <_<

**Comment by tjclement** [comment]
Even though it is the same as running make monitor from the firmware directory, I think it's nice that we have these scripts in the root that make building and testing easy for newcomers. Therefore, I would like monitor.sh (and all other shell files) to stay as they are.

**Comment by Cat-Ion** [comment]
Absolutely. But there's another file (`monitor`) that has exactly the same contents.

Well, had. Renze deleted it a few days ago, so now the script isn't redundant any more...

**Comment by tjclement** [comment]
Ah I see. That gets created if you run make monitor from the root directory. I guess it must have been committed by accident by someone. Thanks for reporting!

**Comment by Cat-Ion** [comment]
Ahh. That explains it. I had wondered how this happened. Silly implicit rules...

**Pull request closed without merging**